src/main/src/main/java/org/geoserver/catalog/ResourcePool.java

```diff
@@ -5,7 +5,7 @@
  */
 package org.geoserver.catalog;
 
-import java.awt.*;
+import java.awt.RenderingHints;
```

Comment:
?

Reply:
will revert, that's probably eclipse being smart